src/Gui/Application.cpp

```diff
@@ -689,6 +689,35 @@ void Application::slotNewDocument(const App::Document& Doc)
 #if QT_VERSION < 0x050000
     qApp->processEvents(); // make sure to show the window stuff on the right place
 #endif
+
+    ParameterGrp::handle hGrp = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/View");
+    std::string newDocView = hGrp->GetASCII("NewDocumentCameraOrientation","Axonometric");
+    if (newDocView == "Use current"){
+        newDocView = "Axonometric"; //failed to set via python, likely had no open document at the time
```

Comment:
I wonder if the default shouldn't be "Top", so for existing users nothing will change until they set the property...

Reply:
I'll see if I can set default to Top view.  I think it would inconvenience too many existing arch/draft users who are depending on the Top view to be default because they have working plane set to auto.